restapi/src/test/java/io/stargate/web/docsapi/resources/DocumentResourceV2Test.java

```diff
@@ -45,151 +45,10 @@
   private Method wrapResponseIdPageState;
 
   @Before
-  public void setup() throws NoSuchMethodException {
+  public void setup() {
     documentResourceV2 = new DocumentResourceV2();
     Whitebox.setInternalState(documentResourceV2, DocumentService.class, documentServiceMock);
     Whitebox.setInternalState(documentResourceV2, Db.class, dbFactoryMock);
-
-    wrapResponseId =
-        DocumentResourceV2.class.getDeclaredMethod("wrapResponse", JsonNode.class, String.class);
-    wrapResponseId.setAccessible(true);
-    wrapResponseIdPageState =
-        DocumentResourceV2.class.getDeclaredMethod(
-            "wrapResponse", JsonNode.class, String.class, String.class);
-    wrapResponseIdPageState.setAccessible(true);
-  }
-
-  @Test
-  public void wrapResponse_nodeIdDefined()
```

Comment:
Mm I think unit tests aren't running in CI...this didn't run properly because `wrapResponse` was removed in a previous PR.

Reply:
It's because it uses JUnit 4, which is not supported in SG now

Reply:
So try converting to JUnit 5 :)

Reply:
oh noooooo

alright - I'm not sure if powermock is supported by junit 5 but I sure hope so

Reply:
I'll take a look at this tomorrow

Reply:
powermock should be avoided in the first place IMHO :)